### PR TITLE
provide possibility to set up data dir

### PR DESCRIPTION
**Pull request by btomala**
The changes which I requested in issue https://github.com/scoverage/sbt-scoverage/issues/261 

I hope it will be helpful.

### CODE REVIEW

**Comment by btomala** [comment]
This should solve the issue https://github.com/scoverage/sbt-scoverage/issues/224#issuecomment-312169010 by the setting up data directory.

**Comment by btomala** [comment]
@gslowikowski I will glad if you can consider merging this feature as well.

**Comment by gregghz** [comment]
Will this ever be merged?

**Comment by zabbal** [comment]
I'm interested as well - what's blocking the merge?

File: src/main/scala/scoverage/ScoverageSbtPlugin.scala [hunk 1]
```diff
@@ -77,7 +78,7 @@ object ScoverageSbtPlugin extends AutoPlugin {
         }
         Seq(
           Some(s"-Xplugin:${pluginPath.getAbsolutePath}"),
-          Some(s"-P:scoverage:dataDir:${crossTarget.value.getAbsolutePath}/scoverage-data"),
+          Some(s"-P:scoverage:dataDir:${coverageDataDir.value.getAbsolutePath}/scoverage-data"),
```
Comment (tfreiling-livongo) [inline src/main/scala/scoverage/ScoverageSbtPlugin.scala:81]: Probably remove the /scoverage-data part, as this can be set by the dataDir

**Comment by btomala** [comment]
If someone is interested in this can take it over.

**Comment by pzhong-livongo** [comment]
I just created a new PR where it's rebasing the latest master.

**Pull request closed without merging**